README.md

```diff
@@ -71,22 +71,23 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+A model method because I don't know what helper methods do and because models are the middle-man between controllers and the database. If a song is being referenced from the database then it makes most sense for the model to run the method that applies the coloring.
 ```
```

Comment:
It's actually a helper method! A good way to differentiate helper and model methods...
* Model methods influence how information is retrieved and put into a database. They operate between the database and the Rails model.
* Helper methods influence how information is displayed in the browser (i.e., how the HTML is rendered). They operate between the controller and the view.